### DISCUSSION TITLE
The folder path in shell prompt doesn't have a color background

### DISCUSSION THREAD

**Discussion by ningji** [Eugeny/tabby#6347] [Q&A]
see the pic, how to change the background for ~/tmp to be the solid blue, and foreground to be black.
basically swap background and foreground colors for each folder in the path ?
![Screenshot from 2022-05-03 17-50-32](https://user-images.githubusercontent.com/5289380/166572516-d21c0270-5116-4ece-ab76-258c44a92132.png)

**Comment by Eugeny** [comment]
Depends on the shell - you'll need to write it as a function and call it from your `$PS1`

**Reply by ningji** [reply]
> Depends on the shell - you'll need to write it as a function and call it from your `$PS1`

i c, thought it came by default. I'm using bash, is there an example i can follow ? thx !

**Reply by Eugeny** [reply]
If you just want to spice up your prompt, try zsh + https://ohmyz.sh/, otherwise try writing up a colorizer like that as a separate script first, and then embed it into your prompt definition.

**Reply by ningji** [reply]
i'll stay with bash for my work, but thx for the info !
